test/e2e/network_test.go

```diff
@@ -830,7 +830,8 @@ func (tc *testContext) createJob(name, image string, command []string, runtimeCl
 // deleteJob deletes the job with the given name
 func (tc *testContext) deleteJob(name string) error {
 	jobsClient := tc.client.K8s.BatchV1().Jobs(tc.workloadNamespace)
-	return jobsClient.Delete(context.TODO(), name, metav1.DeleteOptions{})
+	propPolicy := metav1.DeletePropagationBackground
```

Comment:
This is fine as long as we are collecting the logs.
By leaving them Ophaned the logs are picked up by must gather-extra.

please confirm the logs are collected on job success and failure.

Reply:
you're right, they wouldn't be collected if they were deleted in the background. Changed to orphaned. :+1: